lobsterpy/cli.py

```diff
@@ -915,8 +915,17 @@ def run(args):
                 filename = filename.with_name(zpath(filename.name))
             options = {"are_cobis": False, "are_coops": False}
 
+        struture_filename = args.structure.parent / "POSCAR"
```

Comment:
Can we have a fallback option "POSCAR.lobster". I think the latest Lobster supports this now.

Reply:
I think it is better if we do this in another PR, we do not yet support the newer LOBSTER version or also tested it. But it is good idea though!